### ISSUE TITLE
Image field existing file dropdown select is duplicated when "Toggle All Thubmnails" is clicked multiple times

### ISSUE THREAD

**Issue by mshou** [FLEXIcontent/flexicontent-cck#366]
Hello,

I'm using FLEXIcontent 3.0.9-rc4.
When clicking 3 times in "toggle all tumbnails" a second dropdown appears.

![image](https://cloud.githubusercontent.com/assets/15908034/11243752/adb77a0e-8e01-11e5-9d6d-2dd5645197b8.png)

**Comment by ggppdk** [comment]
Hello please try rc5, this could be because of chosen JS ?

**Comment by mshou** [comment]
Hi,
Upgraded and continue the same issue.

**Comment by ggppdk** [comment]
- please examine the 2nd one with browser's DOM inspector, or with a tool like firebug
is it a box add by "chosen" ?

**Comment by mshou** [comment]
The select hidden by select2 plugin after clicking 3 times on "Toggle All Thumbnails" css is changed from display:none to display:inline-block.

This happens in the backend, I havent' tested in frontend.

**Comment by ggppdk** [comment]
Does it happen in all browsers?

**Comment by mshou** [comment]
Yes.

The second select box:
![image](https://cloud.githubusercontent.com/assets/15908034/11267733/fb7b9a4c-8ea4-11e5-8031-91cbb400d598.png)

**Comment by ggppdk** [comment]
if you open browser console (ctrl-shift-k) or firebug or similar
do you get a JS error when clicking the "toggle" button ?

**Comment by mshou** [comment]
Nope.

**Comment by ggppdk** [comment]
maybe i remember this , 
i think it can be because of the opening effect added recently

i think we should disable the effect 
and show / hide the box instantly

test this:
inside the image field
plugins/flexicontent_fields/image/image.php
```
replace:
obj_disp_toggle.show('slide');
with:
obj_disp_toggle.show();

replace:
obj_disp_toggle.hide('slide');
with:
obj_disp_toggle.hide();
```

**Comment by mshou** [comment]
Tested, and the second box steel appearing..

Is "Toogle All Tumbnails" button and not the "Toogle Image Selector"

**Comment by ggppdk** [comment]
aaahh, i missed the "Toogle All Thumbnails", maybe i read your initial message too quickly, i was thinking you said "Toogle Image Selector", my mistake

yes i confirm the issue, will make a fix

**Comment by ggppdk** [comment]
Fixed with 6913d3f
